src/common/Card.js

```diff
@@ -8,7 +8,7 @@ const styles = css`
     padding: 14px 22px;
     position: relative;
     margin-top: 24px;
-    height: 100px;
+    min-height: 100px;
```

Comment:
it was lost in discussion in #29  but instead of flexible height, the text should be cut with `...` at the end so the card has always the same height

according to this doc:
 
 https://paper.dropbox.com/doc/Knowledgebase-additional-features-Smart-FAQ-1.0.-JB8W3H8ptSG5LCvb7jtuJ

these should be length limits:
```
​Categories:
​​Headings: 45 characters
Perex: 90 characters
​​
​​Articles:
​​Headings: 90 characters
Perex: 100 characters
```

Reply:
hmm, ok, I can do that. What do you think about using a CSS ellipsis overflow instead, though? That way it's possible to define the maximum length (and/or) with CSS and have an '...' if it's too long. It seems more stable to me, as it can maximize the number of characters that can be displayed, without the possibility of an overflow. Limiting at a fixed number of characters might still cause an overflow if the characters are unusually wide, or else, we might up wasting screen space (cutting of characters when there's plenty of space left)

Reply:
I undid the height thing, and implemented the header thing the `text-overflow: ellipsis` way.